.github/workflows/zfs-tests-functional.yml

```diff
@@ -55,12 +58,9 @@ jobs:
             sudo udevadm control --reload-rules
           fi
         fi
-        # Workaround to provide additional free space for testing.
-        #   https://github.com/actions/virtual-environments/issues/2840
-        sudo rm -rf /usr/share/dotnet
-        sudo rm -rf /opt/ghc
-        sudo rm -rf "/usr/local/share/boost"
-        sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+    - name: Report disk space
```

Comment:
I agree it is not crucial to every run. It costs <1s and since there were unexpected issues with the disk space in the runners last year I see little a bit of value in monitoring free space. Might turn out to be better than adding debug prints and re-running the workflow when things go south.

Reply:
> since there were unexpected issues with the disk space in the runners last year

Didn't remember it since initial cleanup additions, but ok, we can clean them up any time.